source/backend/core/Extensions/QueryableExtensions.cs

```diff
@@ -2,6 +2,7 @@
 using System.Linq;
 using System.Linq.Expressions;
 using System.Reflection;
+using DocumentFormat.OpenXml.Office2010.Excel;
```

Comment:
I get this sometimes too, but I'm pretty sure this is not needed.

Reply:
weird, i've cleansed up other instances of it.